### ISSUE TITLE
how to down grade back to 4.1?

### ISSUE THREAD

**Issue by thegeoplayernoob** [ninXout/Crystal-Client#33]
Im having issues with the mod because there isnt too many mods on it so im looking forward to downgrade to 4.1. Is it possible?

**Comment by ghost** [comment]
Yes, you can get the .geode of the mod in releases. Then, you can go into the geode files (Open the geode menu and click on the folder) and then put it in there.

**Comment by thegeoplayernoob** [comment]
Just a question where to put it?

**Comment by ghost** [comment]
Put it in the mods folder inside of the folder it brings you to when you click the folder icon in the geode menu.

**Comment by thegeoplayernoob** [comment]
But then it just says it doesn't load properly

**Comment by ghost** [comment]
Yeah, I just remembered that since 4.0.1 isn’t made for 2.2, it won’t work.

**Comment by ghost** [comment]
W H Y (newest alpha)
![20240303091359_1](https://github.com/ninXout/Crystal-Client/assets/120387312/d3bc3287-dde1-4867-a5eb-821e455c1ac2)

**Comment by ghost** [comment]
Update to 2.204

**Comment by ghost** [comment]
I AM ON 2.204
WIN 10
